packages/atoms/src/SelectItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { FlagIcon, FlagIconName } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Radio } from './Radio';
+import { Text } from './Text';
+
+export type SelectItemProps = {
+    label: string;
+    value: string | number;
+    onSelect: () => void;
+    isSelected: boolean;
+    isLastChild?: boolean;
+    iconName: FlagIconName;
+};
+
+type SelectItemStyleProps = {
+    isLastChild: boolean;
+    isSelected: boolean;
+};
+const selectItemStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+}));
+
+const underlineSectionStyle = prepareNativeStyle<SelectItemStyleProps>(
+    (utils, { isLastChild, isSelected }) => ({
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        flex: 1,
+        color: utils.colors.gray700,
+        paddingVertical: 20,
+        marginLeft: 10,
+        borderBottomWidth: utils.borders.widths.small,
+        borderColor: utils.colors.gray300,
+        extend: [
```

Comment:
nit: I asked Dan about this `extend` for simple conditions earlier and the response was like this:

![Snímek obrazovky 2022-06-03 v 12 32 45](https://user-images.githubusercontent.com/2227592/171837888-9b6d0742-59ee-48a5-beee-d6e3b06e104e.png)

Just FYI.

Reply:
I think we can keep this, in the other hand, now you have nice array of possible states of select... I don't really have strong opinion about this

Reply:
I think if using extend already, it doesn't make too much difference. If I wouldn't use extend, I would just use simple ternary. Maybe I wouldn't make hard rule for this and just go by what seems best for each use case. Viz Dan comment below, I would leave it if you're okay with this :)